style.css

```diff
@@ -494,3 +494,253 @@ input {
   color: #fff;
   z-index: 2;
 }
+
+.project-details-modal {
+  display: none;
+}
+
+.project-details-modal.show {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  background-color: rgb(96, 112, 255, 0.2);
+  backdrop-filter: blur(5px);
+  min-width: 100%;
+  min-height: 100%;
+  top: 0;
+  left: 0;
+  text-decoration: none;
+  color: black;
+  z-index: 2;
+}
+
+.modal-content {
+  background-color: #fff;
+  border-radius: 10px;
+  width: 50%;
+  height: 80%;
+  margin-bottom: 100px;
+  padding: 1rem 1rem 3rem 1rem;
+  flex-wrap: wrap;
+}
+
+.pop-up-topic {
+  font-size: 80px;
+  font-weight: 200;
+}
+
+.pop-up-list {
+  padding-bottom: 6px;
+  font-size: small;
+}
+
+.pop-up-img {
+  width: 100%;
+  height: 300px;
+}
+
+.modal-header {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+}
+
+.pop-up-details-container {
+  display: flex;
+  flex-direction: row;
+  gap: 2em;
+  justify-content: center;
+  flex-wrap: nowrap;
+  margin-top: 2rem;
+}
+
+.pop-up-text {
+  flex: 2;
+  flex-wrap: nowrap;
+  font-size: 9px;
+  gap: 2rem;
+  line-height: 14px;
+}
+
+.pop-tech {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  list-style-type: none;
+}
+
+.pop-up-tech {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  flex-wrap: wrap;
+  list-style-type: none;
+  color: #6070ff;
+  position: relative;
+}
+
+.pop-items {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 24px;
+  font-size: 16px;
+}
+
+.canopy-pc {
+  list-style-type: none;
+  color: black;
+  font-size: 16px;
+}
+
+.pop-up-tech-type {
+  background-color: #ebebff;
+  padding: 4px 12px;
+  border-radius: 8px;
+  margin: 4px 4px 0;
+  font-size: 12px;
+}
+
+.pop-up-button {
+  display: flex;
+  width: 100%;
+  gap: 1rem;
+  margin-top: 2rem;
+}
+
+.pop-button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 14px;
+  gap: 5px;
+  padding: 6px 6px;
+  color: #6070ff;
+  background-color: #fff;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  width: max-content;
+}
+
+.pop-button-source {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 14px;
+  gap: 5px;
+  padding: 6px 6px;
+  color: #6070ff;
+  background-color: #fff;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  width: max-content;
+}
+
+.mobile-project-modal {
+  display: none;
+}
+
+.mobile-project-modal.show {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  background-color: rgb(96, 112, 255, 0.2);
+  backdrop-filter: blur(5px);
+  min-width: 100%;
+  min-height: 100%;
+  top: 0;
+  left: 0;
+  text-decoration: none;
+  color: black;
+  z-index: 2;
+}
```

Comment:
- [ ] Please always try to make your design as close as possible to provided design, if you look at the below-attached table of design comparison, your design needs some adjustments related to spaces around the popUp. I would request you kindly fix your PopUp issue, and make it looks professional and similar to Figma's design.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Mobile Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-30 21-31-56](https://github.com/NoelLincoln/mportfolio/assets/66476329/8b39c374-0973-4b4e-ae70-70558d910358)

</td>
<td>

![Screenshot from 2023-07-01 00-10-17](https://github.com/Kwamay/Porfolio_Setup_Project/assets/66476329/70feb650-0f80-4b5c-8db9-a2bd011a3b34)

</td>
</tr>
</table>

### Desktop Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-30 21-39-57](https://github.com/NoelLincoln/mportfolio/assets/66476329/eaed1c3c-b6ae-4536-9437-9f72f2d0eb8b)

</td>
<td>

![image](https://github.com/Kwamay/Porfolio_Setup_Project/assets/66476329/81668a6a-5a07-4474-9277-6ec426ac483b)

</td>
</tr>
</table>